recipe/meta.yaml

```
@@ -1,41 +1,51 @@
 {% set name = "pyuca" %}
 {% set version = "1.2" %}
-{% set hash_type = "sha256" %}
-{% set hash = "8a382fe74627f08c0d18908c0713ca4a20aad5385f077579e56208beea2893b2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash }}
+  sha256: 8a382fe74627f08c0d18908c0713ca4a20aad5385f077579e56208beea2893b2
+  patches:
+    - unicode_license.patch
 
 build:
-  noarch: python
   number: 1
```

```suggestion
  number: 0
```
For new packages, we generally reset the build number

---

> When forking a new feedstock, we should try to keep the build number. Not doing that would cause conda-forge's package to be considered before our package if both defaults and conda-forge are mixed.
> -JC

Need some standardization on this front

---

There general rule of thumb is to try to not unnecessarily diverge from conda-forge. The build number will eventually diverge, but we don't need to diverge right at the beginning.